ivy/data_classes/array/experimental/creation.py

```diff
@@ -264,3 +264,31 @@ def mel_weight_matrix(
             lower_edge_hertz,
             upper_edge_hertz,
         )
+
+    def unsorted_segment_mean(
+            self: ivy.Array,
+            segment_ids: ivy.Array,
+            num_segments: Union[int, ivy.Array],
+    ) -> ivy.Array:
+        r"""
+        ivy.Array instance method variant of ivy.unsorted_segment_mean. This method
+        simply wraps the function, and so the docstring for ivy.unsorted_segment_mean
+        also applies to this method with minimal changes.
+        Parameters
+        ----------
+        self
+            The array from which to gather values.
+        segment_ids
+            Must be in the same size with the first dimension of `self`. Has to be
+            of integer data type. The index-th element of `segment_ids` array is
+            the segment identifier for the index-th element of `self`.
+        num_segments
+            An integer or array representing the total number of distinct segment IDs.
+        Returns
+        -------
+        ret
+            The output array, representing the result of a segmented mean operation.
+            For each segment, it computes the mean of values in `self` where
+            `segment_ids` equals to segment ID.
+        """
```

Comment:
docstring missing Examples

Reply:
sorry accidentally seemed to have closed this, could you please add examples here?

Reply:
Done.

Reply:
![image](https://github.com/unifyai/ivy/assets/52187856/4f5e391c-5086-43f7-8b89-859ace606279)
docstring example isn't matching actual results

Reply:
sorry, updated the example output.